tools/data-migration-troubleshooting.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Data Migration Troubleshooting
+summary: Learn how to diagnose and resolve issues when you use Data Migration.
+category: tools
+---
+
+# Data Migration Troubleshooting
+
+This document summarizes some commonly encountered errors when you use Data Migration, and provides the solutions.
+
+If you encounter errors while running Data Migration, try the following solution:
+
+1. Check the log content related to the error you encountered. The log files are on the dm-master and dm-worker deployment nodes. You can then view [common errors](#common-errors) to find the corresponding solution.
+
+2. If the error you encountered is not involved yet, and you cannot solve the problem yourself by checking the log or monitoring metrics, you can contact the corresponding sales support staff.
+
+3. After the error is solved, restart the task using dmctl.
+
+    ```bash
+    resume-task ${task name}
+    ```
+
+However, in some cases, you need to [reset the data synchronization task](#reset-the-data-synchronization-task).
```

Comment:
"in some cases"  Could you make it more specific?

Reply:
The link here links to the specific details. I'll update the wording here in another PR.